src/html.rs

```diff
@@ -574,6 +574,23 @@ where
                 if entering {
                     if ncb.info.eq("math") {
                         self.render_math_code_block(node, &ncb.literal)?;
```

Comment:
Arguably, this could be converted to a language-specific renderer that's registered by default.